Add CM `switch_controller` service timeout as parameter to spawner.py (#1790)
ros-controls/ros2_control: controller_manager/doc/userdoc.rst

      -u, --unload-on-kill  Wait until this application is interrupted and unload controller
      --controller-manager-timeout CONTROLLER_MANAGER_TIMEOUT
                            Time to wait for the controller manager
      --switch-timeout SWITCH_TIMEOUT

>                             Time to wait for a successful state switch of controllers. Useful if the controller_manager is not immediately ready, e.g., paused

Reviewer: ```suggestion
                            Time to wait for a successful state switch of controllers. Useful if controllers cannot be switched immediately, e.g., paused
```

.. code-block:: console

    $ ros2 run controller_manager unspawner -h
    usage: unspawner [-h] [-c CONTROLLER_MANAGER] [--controller-manager-switch-timeout CONTROLLER_MANAGER_SWITCH_TIMEOUT] controller_names [controller_names ...]

    positional arguments:
      controller_names      Name of the controller

    options:
      -h, --help            show this help message and exit
      -c CONTROLLER_MANAGER, --controller-manager CONTROLLER_MANAGER
                            Name of the controller manager ROS node
      --controller-manager-switch-timeout CONTROLLER_MANAGER_SWITCH_TIMEOUT
                            Time to wait for the controller manager to switch controllers

``hardware_spawner``
^^^^^^^^^^^^^^^^^^^^^^

.. code-block:: console

    $ ros2 run controller_manager hardware_spawner -h

>     usage: hardware_spawner [-h] [-c CONTROLLER_MANAGER] [--controller-manager-timeout CONTROLLER_MANAGER_TIMEOUT] (--activate | --configure) hardware_component_name

Reviewer 2: Do we have it for the hardware spawner as well?

Author: the timeout for this service call yes

https://github.com/ros-controls/ros2_control/blob/0433960580c5834b11af9703e3e34ace86824d01/controller_manager/controller_manager/hardware_spawner.py#L159-L160

Reviewer 2: Ahh ok ok, this is a different one. LGTM

.. code-block:: console

    $ ros2 run controller_manager unspawner -h
    usage: unspawner [-h] [-c CONTROLLER_MANAGER] [--switch-timeout SWITCH_TIMEOUT] controller_names [controller_names ...]

    positional arguments:
      controller_names      Name of the controller

    options:
      -h, --help            show this help message and exit
      -c CONTROLLER_MANAGER, --controller-manager CONTROLLER_MANAGER
                            Name of the controller manager ROS node
      --switch-timeout SWITCH_TIMEOUT

>                             Time to wait for a successful state switch of controllers. Useful if the controller_manager is not immediately ready, e.g., paused

Reviewer: ```suggestion
                            Time to wait for a successful state switch of controllers. Useful if controllers cannot be switched immediately, e.g., paused
```
